homeassistant/components/mqtt/vacuum/schema.py

```diff
@@ -15,18 +19,23 @@
 )
 
 
-def services_to_strings(services, service_to_string):
+def services_to_strings(
+    services: VacuumEntityFeature | int,
+    service_to_string: dict[VacuumEntityFeature, str],
+) -> list[str]:
     """Convert SUPPORT_* service bitmask to list of service strings."""
-    strings = []
+    strings: list[str] = []
     for service in service_to_string:
         if service & services:
             strings.append(service_to_string[service])
     return strings
 
 
-def strings_to_services(strings, string_to_service):
+def strings_to_services(
+    strings: list[str], string_to_service: dict[str, VacuumEntityFeature]
```

Comment:
For symmetry with the other function, might be worth to add a trailing comma here so black moves all arguments to separate lines.

Reply:
I sometimes wonder about that... but I think either is fine.